new-nextjs-client/pages/index.module.scss

```diff
@@ -1,6 +1,19 @@
 .container {
   padding: 0 2rem;
   background: $main-background;
+  display: grid;
+  grid-template-rows: 7rem auto;
+
+  .topLayout {
+    display: flex;
+    justify-content: space-between;
+    .addYear {
```

Comment:
Let's use `.addYearWrapper`. 

Let's append  "Wrapper" to the className of an element that wraps a component and is used for positioning such as padding/margin